### PR TITLE
Keith Haag - Sprint Challenge 

### PR DESCRIPTION
**Pull request by kkhaag**

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -54,29 +54,58 @@ table {
...
 
 p {
     line-height: 1.4;
+    font-size: 1.5rem;
 }
 
 .container {
-    width: 800px;
+    width: 100%;
+    max-width: 1000px;
```
Comment (HTMLGhozt) [inline css/index.css]: We talked about this previously, why max-width is 800px and the rest is margins. We end up having the paragraphs in `.top-content` breach the wanted 800px.
breach

File: css/index.css [hunk 2]
```diff
@@ -112,9 +142,35 @@ p {
...
+    background: hotpink;
+    }
+
+.middle-content .boxes .box:nth-of-type(8) {
+    background: indigo;
+    }
+
+.middle-content .boxes .box:nth-of-type(10) {
+    background: lawngreen;
+    }
```
Comment (HTMLGhozt) [inline css/index.css]: I am still impressed that you used the pseudo class `:nth-of-type`. As far as I know, no one else did this.

File: css/index.css [hunk 3]
```diff
@@ -104,6 +133,7 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
+    font-size: 1.5rem;
```
Comment (HTMLGhozt) [inline css/index.css]: adding a `min-width` to these boxes would have stopped them from squishing on browser resize.

**Review by indifferentghost — COMMENTED** [review]
Beyond the basics I mentioned I'd just keep classes nested inside media queries indented properly
```
@media (max-width: 420px) {
     .class {
          attribute: value;
     }
     .class2 {
          attribute: value;
     }
}
```

**Pull request closed without merging**